src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Boolean.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    // Boolean(arg:s|n)
+    // Corresponding Excel and DAX function: Boolean
+    internal sealed class BooleanFunction : BuiltinFunction
+    {
+        public const string BooleanInvariantFunctionName = "Boolean";
+
+        public override bool RequiresErrorContext => true;
+
+        public override bool IsSelfContained => true;
+
+        public override bool SupportsParamCoercion => false;
+
+        public BooleanFunction()
+            : base(BooleanInvariantFunctionName, TexlStrings.AboutBoolean, FunctionCategories.Text, DType.Boolean, 0, 1, 1, DType.String)
```

Comment:
arg type here doesn't match the signature on line 11

Reply:
Maybe a checkinvocation overload is needed?

Reply:
Removed |n from the comment. If we want more overloads then we need to design the behavior.